src/components/free-willy/FreeWilly.tsx

```diff
@@ -2,25 +2,33 @@ import LiquidationWithdrawals from 'components/free-willy/LiquidationWithdrawals
 import LiquidationBidChart from 'components/free-willy/LiquidationBidChart';
 import PlaceBid from 'components/free-willy/PlaceBid';
 import MyBids from 'components/free-willy/MyBids';
-import { Container, Stack, Paper } from '@mui/material';
+import { Container, Paper, Grid } from '@mui/material';
 
 export function FreeWilly() {
     return (
-        <Container sx={{maxWidth: '1200px'}}>
-            <Stack spacing={2} sx={{margin: '10px 0px'}}>
-                <Paper elevation={3}>
-                    <LiquidationBidChart/>
-                </Paper>
-                <Paper elevation={3}>
-                    <PlaceBid />
-                </Paper>
-                <Paper elevation={3}>
-                    <MyBids />
-                </Paper>
-                <Paper elevation={3}>
-                    <LiquidationWithdrawals />
-                </Paper>
-            </Stack>
+        <Container sx={{maxWidth: '1200px', padding: '10px'}}>
+            <Grid container spacing={2} >
+                <Grid item xs={12}>
+                    <Paper elevation={3}>
+                        <LiquidationBidChart/>
+                    </Paper>
+                </Grid>
+                <Grid item xs={6}>
```

Comment:
Setting xs to 12 and sm to 6 will make this component responsive.   half screen on small and up, full screen on mobile.  can tweak up to sm/md instead if it looks bad on tablet as well.

Reply:
e.g.
```typescript 
<Grid container>
    <Grid item xs={12} sm={6}>
       //...
    </Grid>
    <Grid item xs={12} sm={6}>
        //...
     </Grid>
</Grid>
```